src/tsickle.ts

```diff
@@ -1489,37 +1504,15 @@ class Annotator extends ClosureRewriter {
           //     Field2 = Field1 + 3,
           // then it's still a constant expression and we inline the constant
           // value in the above branch of this "if" statement.
-          members.set(memberName, member.initializer);
+          // members.set(memberName, member.initializer);
+          this.visit(member.initializer);
         }
       } else {
-        members.set(memberName, i);
-        i++;
+        this.emit(enumIndex.toString());
+        // members.set(memberName, enumIndex);
+        enumIndex++;
```

Comment:
Hilariously, this all works. TS emits something like `A[A["x"] = 1.3] = "x"; A[A["y"] = 2.3] = "y";`, and we follow suit. We'll need an `enumMatches` that takes into account floating point imprecision...